src/Symfony/Bridge/PhpUnit/DeprecationErrorHandler.php

```diff
@@ -50,7 +52,10 @@ public static function register($mode = 0)
             if (false === $mode) {
                 $mode = getenv('SYMFONY_DEPRECATIONS_HELPER');
             }
-            if (DeprecationErrorHandler::MODE_WEAK !== $mode && (!isset($mode[0]) || '/' !== $mode[0])) {
+            if (!in_array($mode, array(
```

Comment:
the in_array + line splits look unreadable to me - what about using two comparisons on a single line instead?